composer.json

```diff
@@ -6,26 +6,25 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.8",
-        "php": ">=7.2"
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9.1"
     },
     "require-dev": {
-        "composer/installers": "~1.5",
-        "drupal/core-composer-scaffold": "^8.8",
+        "composer/installers": "^1.11",
+        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
         "drupal/config_devel": "~1.2",
-        "drush/drush": "~9.0@stable",
-        "guzzlehttp/guzzle": "~6.3",
-        "openeuropa/code-review": "~1.5",
-        "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/task-runner": "~1.0.0-beta5",
-        "phpunit/phpunit": "~6.0",
-        "symfony/dom-crawler": "~3.4"
+        "drupal/drupal-extension": "~4.0",
+        "drush/drush": "^10.3",
+        "openeuropa/behat-transformation-context": "~0.1",
```

Comment:
Seems like we don't need to add this component.
In this component, we don't use behat tests.